controllers/clusterlogging/clusterlogging_controller.go

```diff
@@ -65,10 +67,15 @@ func (r *ReconcileClusterLogging) Reconcile(ctx context.Context, request ctrl.Re
 	telemetry.SetCLMetrics(0) // Cancel previous info metric
 	defer func() { telemetry.SetCLMetrics(1) }()
 
+	removeFinalizer := func(identifier string) error {
```

Comment:
There are `finalizer` utility functions provided by the package `controllerutil`. It might be a better use than defining our own?
[Remove Finalizer](https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.14.5/pkg/controller/controllerutil#RemoveFinalizer)

Reply:
updated internal/k8shandler/finalizer.gogit